indico/modules/categories/client/js/components/CategoryModeration.jsx

```diff
@@ -16,16 +17,26 @@ import {handleSubmitError} from 'indico/react/forms';
 import {useIndicoAxios} from 'indico/react/hooks';
 import {Translate} from 'indico/react/i18n';
 import {indicoAxios} from 'indico/utils/axios';
-import {serializeDate} from 'indico/utils/date';
 import './CategoryModeration.module.scss';
 
+function getEventTime({startDt, endDt}) {
+  startDt = moment(startDt);
+  endDt = moment(endDt);
+
+  if (startDt.isSame(endDt, 'day')) {
+    return `${startDt.format('D MMM YYYY, hh:mm')} → ${endDt.format('hh:mm')}`;
+  } else {
+    return `${startDt.format('D MMM YYYY, hh:mm')} → ${endDt.format('D MMM YYYY, hh:mm')}`;
+  }
```

Comment:
Can we avoid hardcoding date/time formats? This is somewhat messy when it comes to l10n... not all languages use that particular date (or time) format.

Reply:
I changed this to `L` & `LT`